crates/typst/src/math/style.rs

```diff
@@ -300,7 +300,8 @@ pub fn styled_char(styles: StyleChain, c: char, auto_italic: bool) -> char {
                 c,
                 'a'..='z' | 'ı' | 'ȷ' | 'A'..='Z' | 'α'..='ω' |
                 '∂' | 'ϵ' | 'ϑ' | 'ϰ' | 'ϕ' | 'ϱ' | 'ϖ'
-            ),
+            )
+            && variant != Bb,
```

Comment:
What's the reasoning for this? In isolation (for just bb), it seems a bit ad-hoc.

Reply:
The reason was that, if a bb symbol was automatically italized then it wouldn't enter the exception condition, therefore not being rendered properly

Reply:
Maybe we should do auto-italic only for `matches!(variant, Sans | Serif)`. these are the ones that have proper italic variants for all glyphs. 

cc: @Enivex thoughts?

Reply:
So, would it be enough to just change that line to become `matches!(variant, Sans | Serif)`?

Reply:
yeah I think that would make sense

Reply:
done!